src/routes/email/email.controller.ts

```diff
@@ -106,4 +107,13 @@ export class EmailController {
       account: deleteEmailDto.account,
     });
   }
+
+  @Cron(CronExpression.EVERY_WEEK)
+  async deleteUnverifiedEmailsOlderThanAWeek(): Promise<void> {
+    const today = new Date();
+    const oneWeekInMs = 7 * 24 * 60 * 60 * 1_000;
+    const oneWeekAgo = new Date(today.getTime() - oneWeekInMs);
```

Comment:
I'm uncertain whether we should move this into the service. I've included it here as it is alongside the cron time.